src/utils.js

```diff
@@ -389,3 +394,23 @@ export const safeExecute = (
     logger.warn(message, err);
   }
 };
+
+export const getLumigoEventKey = event => {
+  const noCircularEvent = noCirculars(event);
+  return recursiveGetLumigoEventKey(noCircularEvent);
+};
+
+const recursiveGetLumigoEventKey = noCircularEvent => {
```

Comment:
Extract here to other function that recursive get key from an object, and get `LUMIGO_EVENT_KEY` as param, this code we cant resue :)